.github/workflows/workflow.yml

```
@@ -50,18 +48,19 @@ jobs:
     needs:
       # Only publish if other jobs passed
       - test
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Same as above especially as this is only running on tags, we might need 

```suggestion
          fetch-depth: 0
          ref: ${{ github.ref }}
```